_projects/2025/100454764/100454764.Rmd

```diff
@@ -0,0 +1,1052 @@
+              sigma = 8) +
+  theme_void() + 
+  theme(panel.background = element_blank(), 
+        panel.border = element_blank(),     
+        plot.background = element_rect(fill = "#1A1E29",
+                                       color = NA),
+        legend.position = "none") +
+  geom_glowpoint(data = fra_wind, aes(x = longitude, 
+                                      y = latitude, 
+                                      color = primary_fuel,
+                                      size = capacity_mw,
+                                      alpha = 0.3),
+                 shadowsize = 1,
+                 shadowalpha = 0.002) +
+  scale_color_manual(values = fuel_colors) +
+  scale_size_area(max_size = 5) +
+  coord_sf(xlim = c(-6, 10), ylim = c(41, 52)) +
+  annotate("text", x = 2, y = 41.25, label = "WIND", 
+           color = fuel_colors["Wind"], size = 4.7, 
+           family = "audiowide")
+
+## BIOMASS
+
+bio_plants_2.1 <- ggplot() +
+  with_shadow(geom_sf(data = FR, fill = "black", color = "black"),
+              sigma = 8) +
+  theme_void() + 
+  theme(panel.background = element_blank(), 
+        panel.border = element_blank(),     
+        plot.background = element_rect(fill = "#1A1E29",
+                                       color = NA),
+        legend.position = "none") +
+  geom_glowpoint(data = fra_bio, aes(x = longitude, 
+                                     y = latitude, 
+                                     color = primary_fuel,
+                                     size = capacity_mw,
+                                     alpha = 0.3),
+                 shadowsize = 1,
+                 shadowalpha = 0.015) +
+  scale_color_manual(values = fuel_colors) +
+  scale_size_area(max_size = 5) +
+  coord_sf(xlim = c(-6, 10), ylim = c(41, 52)) +
+  annotate("text", x = 2, y = 41.25, label = "BIOMASS", 
+           color = fuel_colors["Biomass"], size = 4.7, 
+           family = "audiowide")
+
+## ARRANGE MAP & TITLE
+
+arranged_maps_2.1 <- total_plants_2.1 + nuclear_plants_2.1 + 
+  hydro_plants_2.1 + oil_plants_2.1 + solar_plants_2.1 + wind_plants_2.1 + 
+  coal_plants_2.1 + bio_plants_2.1 + gas_plants_2.1 + 
+  plot_layout(design = layout)
+
+final_map_2.1 <- arranged_maps_2.1 + plot_annotation(
+  theme = theme(plot.background = element_rect(fill = "#1A1E29",
+                                               color = NA),
+                plot.caption = element_text(color = "white",
+                                            size = 50, 
+                                            hjust = 0.5, vjust = 0.5, 
+                                            margin = unit(c(0.3, 0, 0.3, 0), "cm"),
+                                            family = "audiowide")),
+  caption = "WHAT POWERS FRANCE?")
+
+final_map_2.1
+```
+
+Here is a side by side comparison of the original replication and the alternative version side by side (Need help with this, I don't know why it is not working):
+
+```{r}
+final_map + final_map_2.1 
```

Comment:
This is not working because the second one is a ggplot object, but the first one is already a patchwork object. Try this:

```r
wrap_plots(final_map, final_map_2.1)
```